frontend/web/src/app/services/digital-print/digital-print.service.ts

```diff
@@ -37,6 +37,12 @@ export class DigitalPrintService {
           return data;
       });
   }
+  uploadMagicsFile(digitalPrint: DigitalPrint, magicsFile: any): Observable<string> {
+    return this.client.post(this.endpoint + '/' + digitalPrint.id + '/magics', magicsFile)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)